README.md

```diff
@@ -3,7 +3,7 @@
 `gh` is GitHub on the command line, and it's now available in beta. It brings pull requests, issues, and other GitHub concepts to
 the terminal next to where you are already working with `git` and your code.
 
-![screenshot](https://user-images.githubusercontent.com/98482/73286699-9f922180-41bd-11ea-87c9-60a2d31fd0ac.png)
+![screenshot of gh pr status](https://user-images.githubusercontent.com/98482/84171218-327e7a80-aa40-11ea-8cd1-5177fc2d0e72.png)
```

Comment:
@vilmibm how did you upload this image?

Reply:
Good trick. But for how long does GitHub keeps that image persisted? (I'm questioning if it's safe to use).